oak/server/wasm_node_test.cc

```diff
@@ -37,39 +37,40 @@ std::string DataFrom(const std::string& filename) {
 
 TEST(WasmNode, MalformedFailure) {
   // No magic.
-  ASSERT_EQ(nullptr, WasmNode::Create(nullptr, "test", "", "oak_main"));
+  ASSERT_EQ(nullptr, WasmNode::Create(nullptr, "test", 0, "", "oak_main"));
```

Comment:
Maybe create a constant to avoid this magic number.

Reply:
Left as-is as it's due to be deleted soon.